src/app/components/TxQueue/TxQueueCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { formatEther } from '@ethersproject/units';
+
+import { Body, Box, FromToAccount, Image, LinkApp, TimeElapsed } from '@app/components';
+import { ROUTE_PATHS } from '@app/routing';
+import { selectTransaction, useDispatch } from '@app/store';
+import circleArrow from '@assets/icons/circle-arrow.svg';
+import waiting from '@assets/icons/queue-waiting.svg';
+import { TAddress, TxQueueEntry } from '@types';
+
+export const TxQueueCard = ({ item }: { item: TxQueueEntry }) => {
+  const dispatch = useDispatch();
+
+  const { tx } = item;
+
+  return (
+    <Box py="16px">
+      <Box variant="rowAlign">
+        <Image src={waiting} height="20px" width="20px" mr="8px" />
+        <Body color="PURPLE">WAITING ON ACTION</Body>
+      </Box>
+      <Box variant="rowAlign" mt="2">
+        <FromToAccount sender={tx.from as TAddress} recipient={tx.to as TAddress} />
```

Comment:
Can we change the type of `tx.from` and `tx.to` so these type casts aren't necessary?

Reply:
I would prefer using the `ethers` type for transactions

Reply:
The problem is that we will need to type cast `from` and `to` every time we use it with a function that expects a `TAddress`. We could do declaration merging to change the type in `ethers` itself if we want to keep using the `ethers` type.